tensorflow/compiler/jit/defs.h

```diff
@@ -24,6 +24,7 @@ namespace tensorflow {
 // Name of attribute used to tag operators for compilation with XLA
 extern const char* const kXlaCompileAttr;  // "_XlaCompile"
 extern const char* const kXlaScopeAttr;    // "_XlaScope"
+extern const char* const kXlaAutoJitScopeAttr;  // "_XlaAutoJitScope"
```

Comment:
I think this should have "internal" somewhere in its name to make it obvious that it isn't for end users.  What do you think?

Reply:
Good point. Let's call it `_XlaInternalScope`. It is indeed clearer than `_XlaAutoJitScope` and more distinguishable from _XlaScope.